mkdocs.yml

```diff
@@ -132,6 +133,8 @@ plugins:
       custom_templates: templates
   - mkdocs-opensafely-databuilder
   - macros
+  - multirepo:
+      cleanup: false
```

Comment:
for development only; in production we'd set this to true so it cleans up the imported files after building the site